src/types/usersAttributes.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  Model, InferAttributes, InferCreationAttributes, CreationOptional,
+} from 'sequelize';
+
+interface UsersAttributes extends Model
+<InferAttributes<UsersAttributes>,
+InferCreationAttributes<UsersAttributes>> {
+  id: CreationOptional<number>,
+  fullName: string,
+  email: string,
+  password: string,
+  phone_number?: string | undefined,
+  role: CreationOptional<string>,
+  is_active: CreationOptional<boolean>,
```

Comment:
When creating a new user, sequelize will automatically add `createdAt`, and `updatedAt` fields, for those we're going to need to add them to type 
```suggestion
  is_active: CreationOptional<boolean>,
  createdAt?: CreationOptional<Date>;
  updatedAt?: CreationOptional<Date>;
```

Reply:
done